knip.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://unpkg.com/knip@3/schema.json",
+  "workspaces": {
+    "apps/backend": {
+      "entry": [
+        "src/processes/proc/*.ts",
+        "src/build/bin/*.ts",
+        "src/database/testing/index.ts",
+        "src/graphql/definitions/*.ts"
+      ],
+      "ignore": ["db", "knexfile.js", "src/graphql/__generated__"],
+      "ignoreDependencies": ["pg"],
+      "paths": {
+        "@/*": ["./src/*"]
+      }
+    },
+    "apps/frontend": {
+      "entry": ["src/index.tsx"],
+      "ignore": ["src/gql"],
+      "ignoreDependencies": ["webpack-cli"]
+    },
+    "apps/email": {
+      "entry": ["emails/*.tsx"]
+    }
+  },
+  "webpack": {
+    "config": ["webpack.config.mjs"]
+  },
+  "vitest": {
+    "config": ["vitest.config.ts", "vitest.*.config.ts"],
```

Comment:
Seeing this pattern more often, Knip v3.5.0 uses `vitest*.config.ts` so this could be removed if you update.